assets/src/edit-story/components/panels/document/publish/publish.js

```diff
@@ -204,63 +156,51 @@ function PublishPanel() {
       <PanelContent>
         <PublishTime />
         {capabilities && capabilities.hasAssignAuthorAction && users && (
-          <Row>
-            <FieldLabel id={authorLabelId}>
-              {__('Author', 'web-stories')}
-            </FieldLabel>
-            {isUsersLoading || !visibleOptions ? (
-              <AdvancedDropDown
-                placeholder={__('Loading…', 'web-stories')}
-                disabled
-                primaryOptions={[]}
-                {...dropDownParams}
-              />
-            ) : (
-              <AdvancedDropDown
-                options={queriedUsers}
-                primaryOptions={visibleOptions}
-                searchResultsLabel={__('Search results', 'web-stories')}
-                disabled={isSaving}
-                {...dropDownParams}
-              />
-            )}
-          </Row>
+          <Author />
         )}
-        <Row>
-          {/* @todo Replace this with selection to choose between publisher logos */}
-          <LabelWrapper>
-            <FieldLabel>{__('Publisher logo', 'web-stories')}</FieldLabel>
-            <Required />
-          </LabelWrapper>
-          <MediaWrapper isHighlighted={highlightLogo?.showEffect}>
-            <Media
-              ref={publisherLogoRef}
-              value={publisherLogoUrl}
-              onChange={handleChangePublisherLogo}
-              title={__('Select as publisher logo', 'web-stories')}
-              buttonInsertText={__('Select as publisher logo', 'web-stories')}
-              type={allowedImageMimeTypes}
-              ariaLabel={__('Publisher logo', 'web-stories')}
-              variant={MEDIA_VARIANTS.CIRCLE}
-            />
-          </MediaWrapper>
-        </Row>
         <HighlightRow isHighlighted={highlightPoster?.showEffect}>
```

Comment:
Should this perhaps now be:
```js
        <HighlightRow isHighlighted={highlightPoster?.showEffect || highlightLogo?.showEffect}>
```

Ping @embarks.

Reply:
@barklund that's right, since both inputs share the row now. this logic is just for the "flash" effect